TESTING.asciidoc

```diff
@@ -308,20 +308,19 @@ The REST tests are run automatically when executing the "./gradlew check" comman
 YAML REST tests use the following command (modules and plugins may also include YAML REST tests):
 
 ---------------------------------------------------------------------------
-./gradlew :rest-api-spec:yamlRestTest
+./gradlew :rest-api-spec:yamlRestTest --info
```

Comment:
Why are we adding `--info` logging? This seems unnecessary?

Reply:
Minor preference so that it shows the individual tests that are executing vs. just a test counter counting up. I don't have a strong preference, but have found myself always using --info for rest tests. 

I'll defer to your preference @rjernst and remove the --info.